src/highs_bindings.cpp

```diff
@@ -537,8 +537,8 @@ highs_getColsEntries(Highs* h, HighsInt num_set_entries,
                          py::cast(value));
 }
 
-std::tuple<HighsStatus, HighsVarType>
-highs_getColIntegrality(Highs* h, HighsInt col) {
+std::tuple<HighsStatus, HighsVarType> highs_getColIntegrality(Highs* h,
```

Comment:
Looks like I messed up the formatting... I believe VSCode might have done that automatically, let me check how to revert